builders/interoperability/xcm/xc-registration/.pages

```diff
@@ -3,4 +3,5 @@ nav:
   - index.md
   - 'Forum Templates': 'forum-templates.md'
   - 'XC Channel Registration': 'xc-integration.md'
-  - 'XC Asset Registration': 'assets.md'
+  - 'XC Asset Registration via Governance': 'assets.md'
+  - 'Self-Serve Asset Registration': 'foreign-asset-registration.md'
```

Comment:
I'm not that fond of the foreign-asset-registration file name, as it is a bit confusing, but unsure of a better alternative